base/templates/base/public_base.html

```diff
@@ -169,7 +169,9 @@ <h2><a href="{{sidebartitleurl}}" data-ga-category="sidebar-links" data-ga-actio
         </div>
         {% block above_main_content %}{% endblock %}
         <div class="{{content_div_css}}" id="content" role="main">
-            {% if page_type|slugify != 'libnewspage' %}
+	    {% if object_title %}
+	        <h1>{{object_title}}</h1>
```

Comment:
Ugh, yeah; I was looking at this.  It shows up as aligned in my editor.  I eventually figured out that it's because of tabs.  My editor is set up to always indent with spaces, but the original version of this template uses tabs.

Reply:
I think that for now I'm just going to manually tab this and double check that it appears aligned in the GitHub interface.

Reply:
Hmm, upon further inspection it appears there is some other subtle (read: annoying) issue with whitespace that was causing MY editor to insert unwanted tabs.  Jeez.  Anyway, I think this is fixed for now.